win32/getrusage.h

```diff
@@ -72,43 +72,6 @@ struct rusage
 
 	/* Page faults */
 	zend_long ru_majflt;
-#if 0
```

Comment:
Hm, this one is too much as as well. These are prepared things which are not yet supported on Windows. But please let them stay there.